scripts/generate_test_sql/events.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2017-present Kensho Technologies, LLC.
```

Comment:
This seems like a new file. If so, the year has to be the present year -- this is extremely important! Please be very careful with it.